common/src/main/java/org/apache/rocketmq/common/protocol/header/ResetOffsetRequestHeader.java

```diff
@@ -22,12 +22,21 @@
 import org.apache.rocketmq.remoting.exception.RemotingCommandException;
 
 public class ResetOffsetRequestHeader implements CommandCustomHeader {
+
     @CFNotNull
     private String topic;
+
     @CFNotNull
     private String group;
+
+    private int queueId = -1;
+
+    @CFNotNull
+    private Long offset;
```

Comment:
The CFNotNull of offset is not necessary, and will produce WARN for old client

Reply:
ok, I have fixed it